site/docs/community.md

```diff
@@ -86,7 +86,17 @@ Hosts are required to ensure that:
 2. All talks should be vendor-neutral and not sales pitches
 3. Each meetup should have at least two talks with speakers representing different companies/organizations
 4. Planned meetups ought to be brought to the attention of the [dev list](#mailing-lists)
-5. All [Community Guidelines](#community-guidelines) must be respected
+5. The talk selection process is transparent and community members who are interested in speaking can/know how to propose a session
+6. All [Community Guidelines](#community-guidelines) must be respected
+
+Meetups *must* be small events under the ASF branding guidelines (linked above) and are typically small, informal gatherings. If you're unsure whether an event is a meetup, meetups usually:
+
+* Rely on a curated selection of talks, where organizers work with the community to include a diverse representation of speakers, topics, and companies. They don't use a formal call for papers like a conference.
+* Are single-tracked and accommodate only a handful of sessions.
+* Take place over a ~4 hour window.
+* Are hosted/sponsored by 1-3 companies providing food, drinks, and/or *unpaid* meeting space. They don't require exorbitant financial support or selling booth space or marketing opportunities to put on.
```

Comment:
Sorry if this is a duplicate but I don't see my original comment. I think this should be less opinionated and more informational, like the originally proposed wording: "Are sponsored by one or two companies providing food, drinks, or meeting space; not by selling booth space or marketing opportunities". I think you could also work in "substantial financial support" if that's important?

Reply:
Updated with the original language; added "substantial financial support". I still kept it at 1-3 companies, as I think that's fairly standard across what most communities do for their meetups.